cengage/react-magma
# feat(Stepper): Displays step based content for use in multi-step interfaces.

Issue: # [1143](https://github.com/cengage/react-magma/issues/1143)

## What I did

New Stepper component I built with my mitts, used commonly in wizard interfaces.



## Screenshots:
![Steppah](https://github.com/cengage/react-magma/assets/77400920/bce8fbc6-2486-4151-8c7b-648ea12b4f85)

## Checklist 

## How to test

- Verify the Stepper is showing the right values for the current, incomplete, and complete steps.
- Make sure the text doesn't collide between labels.
- Check the responsive states between labels, no labels, and summary.
- Check the regular layout states of labels, no labels, and summary.
- Test the a11y throughout the steps.
- See if 18n works as intended.
